scale/storage/views.py

```diff
@@ -190,7 +162,7 @@ def post(self, request):
         :returns: the HTTP response to send back to the user
         """
 
-        if self.request.version == 'v5':
+        if self.request.version != 'v6':
```

Comment:
Need to support v7; this can be addressed in a future PR

Reply:
We'll add support for v7 in a future PR. Figured it was out of scope for removing the v5 APIs.

Reply:
Good point. Wholeheartedly agree.